src/components/DecorativeHero/DecorativeHero.js

```diff
@@ -0,0 +1,13 @@
+import './_DecorativeHero.scss'
+import template from './DecorativeHero.pug'
+import data from '../../Data/data.json'
+export class DecorativeHero {
+  constructor (node) {
+    this.node = node
+    this.generateHTML()
+  }
+
+  generateHTML () {
+    this.node.innerHTML = template(data[2])
```

Comment:
Try to change the data structure to not rely on the order of the components to get its data.